libraries/lib-effects/PerTrackEffect.cpp

```diff
@@ -320,7 +320,8 @@ bool PerTrackEffect::ProcessPass(TrackList &outputs,
          }
          if (results) {
             const auto t1 = ViewInfo::Get(*FindProject()).selectedRegion.t1();
-            PasteTimeWarper warper{ t1, mT0 + wt.GetEndTime() };
+            PasteTimeWarper warper { t1,
+                                     mT0 + (*results->begin())->GetEndTime() };
```

Comment:
Yes, this corrects a mistake I introduces at af5911aa2e59f1ebcdc0c4e5e8c8e9581fb49dd5